### PR TITLE
docs(puppeteer): update support table

### PR DESCRIPTION
**Pull request by rwaskiewicz**
this commit updates the puppeteer section of the breaking_changes
document to clarify the version of puppeteer that is supported

this commit also updates the support table for puppeteer versions for stencil v3. it serves as the documentation for https://github.com/ionic-team/stencil/pull/3810

Screen Shot 2022-11-10 at 4 38 13 PM

### CODE REVIEW

**Review by tanner-reits — APPROVED** [review]

File: src/docs/reference/support-policy.md [hunk 1]
```diff
@@ -126,8 +126,8 @@ please see [this document](https://github.com/ionic-team/stencil/blob/main/docs/
 
 #### Puppeteer
 
-| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11 | Puppeteer v12 | Puppeteer v13 |
-|:---------------:|:--------------:|:-------------:|:-------------:|:-------------:|:-------------:|
-|       V3        |   &#10060;     |   &#9989;     |   &#9989;     |   &#9989;     |   &#9989;     |
-|       V2        |   &#9989;      |   &#9989;     |   &#10060;    |   &#10060;    |   &#10060;    |
-|       V1        |   &#9989;      |   &#10060;    |   &#10060;    |   &#10060;    |   &#10060;    |
+| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-18 | Puppeteer 19 |
```
Comment (alicewriteswrongs) [inline src/docs/reference/support-policy.md:129]: we've got a little inconsistency here, if we want to stick with what we've been doing maybe this?

```suggestion
| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11-18 | Puppeteer v19 |
```

or we could change them all to `Puppeteer 5-9`, `Puppeteer 10`, etc

**Review by alicewriteswrongs — CHANGES_REQUESTED** [review]
noticed one thing

File: src/docs/reference/support-policy.md (hunk 1 above)
Comment (rwaskiewicz) [reply src/docs/reference/support-policy.md]: @alicewriteswrongs I added 'v' in https://github.com/ionic-team/stencil-site/pull/932/commits/d5f3cea41c422b82c05db24c3930989cf5cb11ac and fixed up the table alignment as a result in that same commit

**Review by alicewriteswrongs — APPROVED** [review]
lgtm!

**Pull request merged**